src/cdk/testing/kagekiri.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+declare module 'kagekiri' {
+  export function querySelectorAll(selector: string, root: Element): NodeListOf<Element>;
+}
```

Comment:
I'm not sure we have a file name convention for this, .spec.d.ts? I added a comment though

Reply:
👍 for a comment. Regarding convention: My concern right now is that this file will end up in the release output since it matches the `.ts` glob. IMO it should be just denoted with `.spec` so that it automatically is filtered out, and it's obvious (without viewing the file) that it's spec only.

Reply:
you mean it will get included because the bazel rule globs on `*.ts`? I think even renaming it to `kagekiri.spec.d.ts` won't fix it, since that won't match the exclude glob `*.spec.ts`. Should I rename it and update the bazel globs in this package?

Reply:
Oh right. I think we can manually exclude the file in the `exclude` glob, or we change it to `.spec.d.ts` and update the glob to handle it. Up to you what way you choose 😄

Reply:
Ok, I called it .spec.d.ts, we now officially have a precedent if we ever need to do this again